package.json

```diff
@@ -58,7 +58,7 @@
 		"jasmine": "^3.6.4",
 		"jasmine-core": "3.4.0",
 		"jasmine-ts": "^0.4.0",
-		"karma": "^6.1.0",
+		"karma": "^6.4.2",
```

Comment:
Why is this changing? I'm always leery when we bump tooling versions in a PR that's not about upgrading tooling :)

Reply:
Good catch! I upgraded it locally when I was debugging, accidentally got it in here.

Reply:
If it works, maybe open a PR that just bumps that... we should try to keep our deps updated, but we should do it intentionally so it's obvious if it ends up causing a failure somewhere down the line :)

Reply:
Pretty sure it's unrelated, I think we are good.